model/jpa/src/main/java/org/keycloak/models/jpa/entities/RealmEntity.java

```diff
@@ -80,6 +80,8 @@
     protected boolean duplicateEmailsAllowed;
     @Column(name="REMEMBER_ME")
     protected boolean rememberMe;
+    @Column(name="USER_DELETE_OWN_ACCOUNT_ALLOWED")
```

Comment:
We used to have columns for every setting, but it doesn't scale very well and requires a lot of updates to DB schema. Hence we introduce the generic realm attributes. Please change this to use realm attributes. In the JSON representation it should be a field, but in the realm entity it should be a realm attribute.